docs/development/source-code-directory-structure.md

```diff
@@ -1,15 +1,13 @@
-# Source code directory structure
+# Source Code Directory Structure
 
-## Overview
-
-The source code of Electron is separated into a few parts, and we are mostly
+The source code of Electron is separated into a few parts, mostly
 following Chromium on the separation conventions.
 
 You may need to become familiar with [Chromium's multi-process
 architecture](http://dev.chromium.org/developers/design-documents/multi-process-architecture)
 to understand the source code better.
 
-## Structure of source code
+## Structure of Source Code
 
 * **atom** - Source code of Electron.
```

Comment:
It sounds to me to switch to the tree structure, I can't find a better way.

Also for expressing tree, we can use the format of `tree` command, which looks better than `|_`:

```
MyApp.app/Contents
├── Info.plist
├── MacOS
│   └── MyApp
└── Frameworks
    ├── MyApp Helper EH.app
    |   ├── Info.plist
    |   └── MacOS
    |       └── MyApp Helper EH
    ├── MyApp Helper NP.app
    |   ├── Info.plist
    |   └── MacOS
    |       └── MyApp Helper NP
    └── MyApp Helper.app
        ├── Info.plist
        └── MacOS
            └── MyApp Helper
```